src/OrchardCore/Orchard.Localization.Abstractions/StringLocalizerExtensions.cs

```diff
@@ -2,19 +2,35 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Linq;
 
 namespace Orchard.Localization.Abstractions
 {
     public static class StringLocalizerExtensions
     {
-        public static LocalizedString Plural(this IStringLocalizer localizer, string name, string pluralText, int count, params object[] arguments)
+        public static LocalizedString Plural(this IStringLocalizer localizer, int count, string singular, string plural, params object[] arguments)
         {
-            if(pluralText == null)
+            if (plural == null)
             {
-                throw new ArgumentNullException(nameof(pluralText), "Plural text can't be null. If you don't want to specify the plural text, use IStringLocalizer without Plural extention.");
+                throw new ArgumentNullException(nameof(plural), "Plural text can't be null. If you don't want to specify the plural text, use IStringLocalizer without Plural extention.");
             }
 
-            return localizer[name, new object[] { new PluralArgument() { PluralText = pluralText, Count = count }, arguments }];
+            return localizer[singular, arguments.Concat(new[] { new PluralArgument() { PluralForms = new[] { singular, plural }, Count = count } }).ToArray()];
+        }
+
+        public static LocalizedString Plural(this IStringLocalizer localizer, int count, string[] pluralForms, params object[] arguments)
```

Comment:
I see what you mean now, but I don't think `PluralArgument` adds anything interesting. I can ask someone in the team if you want a third/fourth opinion.